config.yaml

```diff
@@ -245,15 +245,19 @@ runtime:
     mutexProfileFraction: 0
 
 ## synchronization should be enabled when multiple replicas of agent is deployed
```

Comment:
This statement may be too strong. What about adding "when features for multiple nodes like notification streaming are deployed"?